components/formats-gpl/src/loci/formats/in/InveonReader.java

```diff
@@ -110,7 +110,7 @@ public boolean isThisType(RandomAccessInputStream stream) throws IOException {
   {
     FormatTools.checkPlaneParameters(this, no, buf.length, x, y, w, h);
 
-    int planeSize = FormatTools.getPlaneSize(this);
+    long planeSize = (long) FormatTools.getPlaneSize(this);
```

Comment:
That's up for debate I guess, but outside the scope of this PR.  Either way the return value of ```getPlaneSize(this)``` would need to be cast - if it returns ```long``` we're just casting when a byte array is constructed instead.